crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -907,6 +907,12 @@ type Query {
 	epoch(id: Int): Epoch
 	checkpoint(id: CheckpointId): Checkpoint
 	transactionBlock(digest: String!): TransactionBlock
+	"""
+	The coin objects that exist in the network.
+	The type field is a string of the inner type of the coin
+	by which to filter (e.g., 0x2::sui::SUI).
```

Comment:
Does `0x2::sui::SUI` actually work as a filter, or do you need to canonicalize it before passing it in (for now)?

I don't mind if you have to canonicalize it, initially, but just want to make sure that the doc comment is not suggesting an example that won't work.

Reply:
It also seems like `type` is an optional parameter.  Could you augment the docs to say what happens when you omit it?

Reply:
Final thought (this one we can follow-up on): Typically, with the other connections, (`checkpointConnection` is the odd one out here), we offer a way, with the `Query` version of the connection, to write a generic query -- one that supports any kind of query you could have constructed through some number of nested queries. For `coin` this means introducing a `CoinFilter` that allows folks to optionally supply the address as well as the type.  For `checkpointConnection` it involves adding an optional filter by epoch.

Reply:
I'll do a follow up PR on the `CoinFilter`, that's a good idea.

Reply:
> Does `0x2::sui::SUI` actually work as a filter, or do you need to canonicalize it before passing it in (for now)?
> 
> I don't mind if you have to canonicalize it, initially, but just want to make sure that the doc comment is not suggesting an example that won't work.

It does not work as in the comment. I will update and then try to update it everywhere else. Then we'll have to update it again to mention that we support both shorthand & canonical.